### DISCUSSION TITLE
Weekly Report 11

### DISCUSSION THREAD

**Discussion by LeonmAndreesen** [SE-TINF22B6/time2shine#56] [Show and tell]
Hello everybody, 

currently we are finishing up our work started in Semester 3. We now have a functional website. The website contains features such as a login page, a interface, some links to other subwebsites and we work on a videotrailer. 
![image](https://github.com/SE-TINF22B6/time2shine/assets/123726672/6ad0b542-c849-4cb9-bc09-c6e14bb041d9)
In that screenshot you can the landing page. All the buttons you can see are functional. 
If you like to, you can login via your github account. 
Concerning our features we have the working website. Besides that we have a working database and i interface for playing the game. 
We have a milestone. At the point which when we have finished the first game we will start doing a new game. We can use some bits of the last game for all the other games that are planned. We are planning to do at least one other game

**Comment by mirak55** [comment]
Dear time2shine team,
your website looks awesome at this point, especially with the modern dark design of your landing page. With all your functions to log in and to play the first game which seems to be Black Jack I think you are on a good way to finishing your website. 
I am looking forward to seeing your second game implemented.

**Comment by maxschwinghammer** [comment]
Dear time2shine team,
your website design is really great and makes a professional impression. The integration of the Github login is a good idea that makes it easier for users to access and gives your site a modern touch. I look forward to hearing more about your planned games and the video trailer.

**Comment by MargotGsb** [comment]
Dear Team time2shine,
your wibsite looks very nice on the first impression! I think it is a great idea to put in one other game, in the course of this project. I am really excited for your videotrailer and I am looking forward to see your progress this semester!

Good work and Greetings from Team Plapy

**Comment by Challenger3010** [comment]
Dear time2shine Team,
your website looks very nice and you have achieved a lot so far. We from CookHub look forward to play Black Jack and your other games.
Keep it up!

**Comment by fehlend0** [comment]
Good job! It's great to see your progress in building a functional website with features like a login page and interface. The inclusion of a video trailer sounds exciting too. Keep up the good work!

**Comment by Elizabethstella** [comment]
Great job on the progress! The functional website and game interface are impressive achievements. Looking forward to seeing more!
